.repos/repositories.js

```diff
@@ -91,13 +92,35 @@ const cloneRepository = async (argv) => {
   await Promise.all(promises);
 };
 
+const prepareRepository = async (argv) => {
+  for (const repository of getRepositories(argv)) {
+    console.log(`Preparing repository ${repository.name}...`);
+    await execa('rm', ['-f', `${currentDir}/${repository.destination}/${repository.local_path}/.vuepress`]);
+    await execa('rm', ['-f', `${currentDir}/${repository.destination}/doc/package.json`]);
```

Comment:
Why removing the `package.json` file ?

Reply:
Very good question as this is a pretty weird thing I had to do in order to make the whole build process work. Let me explain.

The aim here is to build a sub-repo (e.g. `core-2`) by linking the `.vuepress` files inside it and passing it to the `vuepress` binary installed in the main repo (i.e. this one).
So, we take the `.vuepress` files from `src/.vuepress` and link them to `.repos/core-2/doc/2/`, and then call `$(npm bin)/vuepress build .repos/core-2/doc/2/`, right?
The weird thing is that vuepress complains about not finding `.repos/core-2/doc/package.json` and I frankly don't undertand why.
The workaround I found is to link `package.json` to `.repos/core-2/doc/package.json` - and it's working all right. For this reason, before linking it, I `rm -f` a hypothetic ` .repos/core-2/doc/package.json` file that might have been put there before, so that the `ln -s` doesn't fail.

Hope this clarifies the point.